### PR TITLE
work-in-progress: adding new featured section to the frontpage. 

### PR DESCRIPTION
**Pull request by drewrwilson**
related to #151

### CODE REVIEW

**Comment by ScottDowne** [comment]
hey @drewrwilson I'll be the one looking at this when it's done :)

Feel free to ping me when you think it's ready for me to take a look.

File: src/pages/home.js [hunk 1]
```diff
@@ -44,13 +44,13 @@ module.exports = React.createClass({
...
-              buttonHref="https://www.youtube.com/watch?v=I-UPyuW5j9o&feature=youtu.be"
-              buttonLabel="watch video"
+              src1x="/assets/digital-tools-for-activism.jpg"
+              src2x="/assets/digital-tools-for-activism@2x.jpg"
+              label="Tools for Activism"
+              buttonHref="https://github.com/mozilla/toolsforactivism"
+              buttonLabel="see the list"
             >
-              In 2015, we fought to keep the Internet free of fast lanes, slow lanes and corporate control. Our community petitioned Congress and made a difference.
+              Whether we're talking about human rights or climate change, in the 21st Century all campaigns will be won and lost on the Internet. The Open Web allows us to organize, coordinate, communicate, mobilize and take collective action.
```
Comment (brettgaylor) [inline src/pages/home.js]: "From climate change to human rights, 21st century campaigns will be won and lost online.  Check out this list of tools from Mozilla and our friends that help us organize, coordinate, communicate, mobilize and take collective action.

**Comment by drewrwilson** [comment]
@ScottDowne I think this is ready to be merged in now. @brettgaylor confirm?

**Comment by brettgaylor** [comment]
👍 merge away, @ScottDowne !

**Comment by ScottDowne** [comment]
Awesome! I'm in transit but I'll take a look in a bit :)

**Comment by ScottDowne** [comment]
Landed, should be on staging any minute, which can be seen here: https://advocacy-mozilla-org-staging.herokuapp.com/en-US

@brettgaylor @drewrwilson when should we push this to production? I can do tomorrow at the earliest.

**Comment by drewrwilson** [comment]
Looks good!

**Comment by drewrwilson** [comment]
Thanks, @ScottDowne! 🏄

**Comment by ScottDowne** [comment]
No prob, and thank you for the patch :cake:  :rocket:

**Comment by ScottDowne** [comment]
FYI this went to production today: https://advocacy.mozilla.org/en-US/

**Pull request merged**